### PR TITLE
CentOS/RHEL 7 test fixes

### PR DESCRIPTION
**Pull request by vojtechtrefny**
These changes allow running dbus tests on CentOS/RHEL 7 using Python 2.

### CODE REVIEW

File: src/tests/dbus-tests/test_50_block.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def test_open(self):
         self.assertIsNotNone(dbus_fd)
 
         fd = dbus_fd.take()
-        mode = fcntl.fcntl(fd, fcntl.F_GETFL) & os.O_ACCMODE
+        mode = fcntl.fcntl(fd, fcntl.F_GETFL) & 3  # os.O_ACCMODE
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_50_block.py]: Would be nicer to try to import the constant and define it on our own if the import fails somewhere above.

File: src/tests/dbus-tests/test_60_partitioning.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import dbus
 import os
+import re
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_60_partitioning.py]: Some left-over import?

Comment (vpodzime) [inline src/tests/dbus-tests/test_60_partitioning.py]: Also, I don't see how this is using ``fdisk``.

File: src/tests/dbus-tests/test_30_iscsi.py [hunk 3]
```diff
@@ -195,6 +195,12 @@ def test_login_mutual_auth(self):
 
     def test_session(self):
         manager = self.get_object('/Manager')
+
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_30_iscsi.py]: ...*don't* support this

File: src/tests/dbus-tests/test_80_filesystem.py [hunk 4]
```diff
@@ -75,7 +75,7 @@ def test_label(self):
         self.assertIsNotNone(disk)
 
         # create filesystem with label
-        label = 'test'
+        label = 'TEST' if self._fs_name == 'vfat' else 'test'  # XXX udisks changes vfat labels to uppercase
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_80_filesystem.py]: So is it udisks or older mkfs.vfat doing this?

File: src/udiskslinuxpartitiontable.c [hunk 5]
```diff
@@ -350,6 +350,8 @@ udisks_linux_partition_table_handle_create_partition (UDisksPartitionTable   *ta
           goto out;
         }
     }
+  else
```
Comment (vpodzime) [inline src/udiskslinuxpartitiontable.c]: *GError* (in the commit message)

File: configure.ac [hunk 6]
```diff
@@ -433,6 +433,7 @@ AM_CONDITIONAL(HAVE_ZRAM, [test "$have_kbd" = "yes" -a "$have_swap" = "yes"])
 
 # LSM module
 have_lsm=no
```
Comment (vpodzime) [inline configure.ac]: *...only one **of the**...*

File: src/tests/dbus-tests/test_20_LVM.py [hunk 7]
```diff
@@ -13,6 +13,7 @@ class UdisksLVMTest(udiskstestcase.UdisksTestCase):
     def setUpClass(cls):
```
Comment (vpodzime) [inline src/tests/dbus-tests/test_20_LVM.py]: *breaks*

File: src/tests/integration-test [hunk 8]
```diff
@@ -1746,6 +1754,12 @@ if __name__ == '__main__':
                            help='name of test class or method (e. g. "Drive", "FS.test_ext2")')
     cli_args = argparser.parse_args()
 
+    distro, version = get_distro_version()
+    if distro in ('centos', 'enterprise_linux'):
+        print('Skipping integration tests on "%s %s".'\
+              'SCSI debug devices causing kernel panic.' % (distro, version))
+        sys.exit(1)
```
Comment (vpodzime) [inline src/tests/integration-test]: I'm afraid this has to return ``0``, otherwise the tests will be always marked as failed.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Pull request merged**